docs/templates/preprocessing/text.md

```diff
@@ -25,10 +25,34 @@ keras.preprocessing.text.one_hot(text, n,
 
 One-hot encode a text into a list of word indexes in a vocabulary of size n.
 
+This is a wrapper to the `hashing_trick` function using `hash` as the hashing function.
+
 - __Return__: List of integers in [1, n]. Each integer encodes a word (unicity non-guaranteed).
 
 - __Arguments__: Same as `text_to_word_sequence` above.
     - __n__: int. Size of vocabulary.
+    
+## hashing_trick
+
+```python
+keras.preprocessing.text.hashing_trick(text, n,
+                  hash_function=None,
+                  filters='!"#$%&()*+,-./:;<=>?@[\\]^_`{|}~\t\n',
```

Comment:
These arguments should be documented, too. Also please fix the indent in the signature.